client/src/app/pages/sbom-list/sbom-toolbar.tsx

```diff
@@ -35,12 +35,19 @@ export const SbomToolbar: React.FC<SbomToolbarProps> = ({ showFilters }) => {
   return (
     <Toolbar {...toolbarProps} aria-label="sbom-toolbar">
       <ToolbarContent>
-        {showFilters && <FilterToolbar {...filterToolbarProps} />}
-        <ToolbarItem>
-          <Button variant="primary" onClick={() => navigate(Paths.sbomUpload)}>
-            Upload SBOM
-          </Button>
-        </ToolbarItem>
+        {showFilters && (
```

Comment:
It would make sense to have a showUploadButton or equivalent in order to follow the same logic as showFilters. Alternatively the upload button could be separate from the SbomToolbar.

Reply:
@carlosthe19916, 

If the toolbar contains features such as filter, action buttons and pagination then the optional features need to have a conditional to enable/disable them. 
It would therefore make sense to have a component property to control each optional feature.
I believe we have to keep in mind that we might want to refactor that one day and make the toolbar even more generic, to handle sbom, advisories, etc.